tasks/optional-label-check.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: optional-label-check
+spec:
+  params:
+    - name: sourceImage
+      description: the fully qualified image name
+      default: quay.io/cvpops/test-index:v4.10
+
+  steps:
+    - name: inspect-image
+      image: quay.io/rsaar/hacbs-test:latest
+      script: "skopeo inspect docker://$(inputs.params.sourceImage) > /tekton/home/image_inspect.json"
+
+    - name: optional-labels-check
+      image: quay.io/rsaar/hacbs-test:latest
```

Comment:
This is pointing towards your personal repo in quay. We may want to point to the`redhat-appstudio`. So maybe we should merge that PR and push a new build up to quay.io